daemons/gptp/common/ieee1588port.cpp

```diff
@@ -230,8 +230,11 @@ bool IEEE1588Port::init_port(int delay[4])
 void IEEE1588Port::startPDelay() {
 	if (automotive_profile) {
 		if (log_min_mean_pdelay_req_interval != PTPMessageSignalling::sigMsgInterval_NoSend) {
+			long long unsigned int waitTime;
+			waitTime = ((long long) (pow((double)2, log_min_mean_pdelay_req_interval) *  1000000000.0));
```

Comment:
is log_min_mean_pdelay_req_interval always a positive integer?
pow() using doubles isn't "cheap".
If 1 << X would work, it might be better.